README.md

```diff
@@ -0,0 +1,53 @@
+# Vouchers4Veggies
+This project is designed and developed by a team of UC Berkeley students through one of [Cal Blueprint](https://calblueprint.org/)'s project teams during the 2022-23 academic year. 
+
+Learn more about [Vouchers 4 Veggies](https://eatsfvoucher.org/) and [Cal Blueprint](https://calblueprint.org/).
+
+---
+## Getting Started
+
+### Prerequisites
+
+Check your installation of `npm` and `node`:
+
+```sh
+node -v
+npm -v
+```
+
+We strongly recommend using a Node version manager like [nvm](https://github.com/nvm-sh/nvm) (for Mac) or [nvm-windows](https://github.com/coreybutler/nvm-windows) (for Windows) to install Node.js and npm. See [Downloading and installing Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+### Installation
+
+1. Clone the repo & install dependencies
+   1. Clone this repo
+        ```sh
+        git clone git@github.com:calblueprint/vouchers4veggies.git
+        ```
+   2. Enter the cloned directory
+        ```sh
+        cd vouchers4veggies
+        ```
+   3. Install project dependencies. This command installs all packages from [`package.json`](package.json).
+      ```sh
+      npm install
+      ```
+2. Set up secrets:
+   1. Retrieve the `.env` file from [Blueprint's internal Notion](https://www.notion.so/calblueprint/Firebase-Environment-Variables-c30a053fba4c47559f9f4944e4962a9f) (access is required) and add it to the root directory of the repo (`vouchers4veggies/.env`).
```

Comment:
It might be more clear if we tell them to create a new `.env` file in the root directory and then copy the variables from our notion into it.

Reply:
Good callout -- I tweaked the language to make this more clear.